src/devices/acurite.c

```diff
@@ -408,13 +412,77 @@ similar RF encoding and data format:
 - 592TXR temperature and humidity sensor
 - 5-n-1 weather station
 - 6045M Lightning Detector with Temperature and Humidity
+- Atlas
 
     CC RR IIII | IIII IIII | pBMMMMMM | pxxWWWWW | pWWWTTTT | pTTTTTTT | pSSSSSSS
     C:2d R:2d ID:12d 1x BATT:1b TYPE:6h 1x ?2b W:5b 1x 3b T:4b 1x 7b S: 1x 7d
 
 @todo - refactor, move 5n1 and txr decoding into separate functions.
 @todo - TBD Are parity and checksum the same across these devices?
         (opportunity to DRY-up and simplify?)
+
+Acurite Atlas:
+
+| Reading           | Operating Range               | Reading Frequency | Accuracy |
+| ---               | ---                           | ---        | ---             |
+| Temperature Range | -40 to 158°F (-40 to 70°C)    | 30 seconds | ± 1°F |
+| Humidity Range    | 1-100% RH                     | 30 seconds | ± 2% RH |
+| Wind Speed        | 0-160 mph (0-257 km/h)        | 10 seconds | ± 1 mph ≤ 10 mph, ± 10% > 10 mph |
+| Wind Direction    | 360°                          | 30 seconds | ± 3° |
+| Rain              | .01 inch intervals (0.254 mm) | 30 seconds | ± 5% |
+| UV Index          | 0 to 15 index                 | 30 seconds | ± 1 |
+| Light Intensity   | to 120,000 Lumens             | 30 seconds | n/a |
+| Lightning         | Up to 25 miles away (40 km)   | 10 seconds | n/a |
+
+The Atlas reports direction with an AS5600 hall effect sensor, it has 12-bit resolution according to the spec sheet. https://ams.com/as5600
+
+Acurite Atlas Message Type Format:
```

Comment:
should mention the differences in message length depending on whether the lightning detector is installed.